sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

```diff
@@ -278,6 +279,14 @@ PS > Set-SPOTenant -SharingDomainRestrictionMode "BlockList" -SharingBlockedDoma
 
 This example enables users to share with all external collaborators except for those on the BlockedDomainList.
 
+### EXAMPLE 16
+
+```powershell
+PS > Set-SPOTenant -EnableVersionExpirationSetting $true
+```
+
+This example enables file version expiration setting.
```

Comment:
This example opts your tenant into public preview of Improved Version History controls feature. Visit [](http://aka.ms/versioning-overview) to learn more about Admin configurations available to manage versions. By opting in, you are accepting the terms of service for version history limits. [Read the terms of service](https://aka.ms/versioning-termsofservice).

Reply:
We sill use the aka.ms link for now, from Reetu, "when the official docs are published I will update the redirect link, and then when things stabilize during preview or prior to GAing we can replace Aka link with the permanent links"